src/module/data/pseudo-documents/power-roll-effects/base-power-roll-effect.mjs

```diff
@@ -45,6 +47,24 @@ export default class BasePowerRollEffect extends TypedPseudoDocument {
 
   /* -------------------------------------------------- */
 
+  /**
+   * Reference to the grandparent item
+   * @type {DrawSteelItem}
+   */
+  get item() {
+    return this.parent?.parent ?? null;
+  }
```

Comment:
This getter already exists as `document`. We can add an alias, sure, if we're dead certain we won't reuse this for non-items.

```suggestion
  get item() {
    return this.document;
  }
```